pkg/pillar/cmd/downloader/dns.go

```diff
@@ -0,0 +1,42 @@
+package downloader
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/lf-edge/eve/pkg/pillar/cast"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	log "github.com/sirupsen/logrus"
+)
+
+// Handles both create and modify events
+func handleDNSModify(ctxArg interface{}, key string, statusArg interface{}) {
+
+	ctx := ctxArg.(*downloaderContext)
+	status := cast.CastDeviceNetworkStatus(statusArg)
+	if key != "global" {
+		log.Infof("handleDNSModify: ignoring %s\n", key)
+		return
+	}
+	log.Infof("handleDNSModify for %s\n", key)
+	if cmp.Equal(ctx.deviceNetworkStatus, status) {
```

Comment:
Do we need this check? PubSub already does this..

Reply:
Prob not, but this one I just moved into a separate file as is, since it was simple to do so. Can we leave it and clean up further next run?

Reply:
Ok.. Is this just existing code moved - om ok with leaving it then..